testing/tests/DevExpress.ui.widgets/popover.tests.js

```diff
@@ -636,6 +636,74 @@ QUnit.module('content positioning', () => {
         }
     });
 
+    QUnit.test('Popover should be positioned correctly on the target right side when target is inside of svg (T891214)', function(assert) {
```

Comment:
What about adding tests for the `position` utils (see [position.tests.js](https://github.com/DevExpress/DevExtreme/blob/20_2/testing/tests/DevExpress.animation/position.tests.js))?

Reply:
This module tests common position utils, etc calculatePosition - it's to common for adding a test for my new code. Besides, my new util is used in the main calculatePosition function, so it affects all of these tests

Reply:
I've add test to another test module, see the changes